release/models/system/openconfig-hashing.yang

```diff
@@ -0,0 +1,336 @@
+        leaf ingress-interface {
+            type boolean;
+            description "Use the ingress subinterface identifired in the
+                calculation of the hash.";
+        }
+    }
+
+    grouping hashing-policy-config {
+        description
+          "Configuration data for hashing policies.";
+
+        leaf name {
+            type string;
+            description
+              "The name of the hashing policy.
+              When a configured user-controlled policy is created by the
+              system, it is instantiated with the same name in the
+              /system/hashing-policies/hashing-policy/name list.";
+        }
+
+        leaf seed {
+            type uint64;
+            description
+                "The seed used to initialize the hash algorithm";
+        }
+
+        leaf algorithm {
+            type string;
+            description
+                "The name of hash algorithm. This algorithm MUST
+                be a supported algorithm";
+        }
+    }
+
+    grouping hashing-policy-top {
+        description
+            "Top level grouping for hashing configuration and operational state
+            data.";
+
+        container hashing-policies {
+            description
+              "Top level container for hashing, including configuration and
+              state data.";
+
+            list hashing-policy {
+                key "name";
+
+                description
+                  "The list of named policies to be used on the device.";
+
+                leaf name {
+                    type leafref {
+                        path "../config/name";
+                    }
+                    description
+                      "References the name of the hashing policy.";
+                }
+                container config {
+                    description
+                      "Configurable items at the global hash policy level.";
+
+                    uses hashing-policy-config;
+                }
+                container state {
+                    config false;
+                    description
+                      "Operational state data at the global hash policy
+                      level.";
+
+                    uses hashing-policy-config;
+                }
+                container hash-field-modes {
+                    description
+                        "Container for specifying inputs to be used when
+                        calculating the hash.";
+                    container config {
+                        description
+                            "Configurable items at the hashing inputs level.";
+                        uses hashing-inputs;
+                    }
+                    container state {
+                        config false;
+                        description "Operational state data at the hashing
+                            inputs level.";
+                        uses hashing-inputs;
+                    }
```

Comment:
Can you help me understand what this operational sate data is?

Reply:
This is a common pattern in OpenConfig where one can read the applied configuration.  In this case, it is a mirror of the /config container.